tests/cron_job_tests/test_offline_app_setup.py

```diff
@@ -0,0 +1,38 @@
+import mock
+from werkzeug.exceptions import HTTPException
+
+from tests.helpers.unittest_base import BaseTestCase
+
+
+class OfflineAppTest(BaseTestCase):
+    def setUp(self):
+        super(OfflineAppTest, self).setUp()
+
+        from rdr_service.offline.main import app, OFFLINE_PREFIX
+        self.offline_test_client = app.test_client()
+        self.url_prefix = OFFLINE_PREFIX
+
+    def test_offline_http_exceptions_get_logged(self):
+        """Make sure HTTPException are logged when thrown"""
+
+        # Need to raise an HTTPException on a cron call, picking an arbitrary one that is easy to mock
+        with mock.patch('rdr_service.offline.main.mark_ghost_participants') as mock_cron_call, \
+                mock.patch('rdr_service.app_util.check_cron', return_value=True),\
+                mock.patch('rdr_service.services.gcp_logging.logging') as mock_logging:
+            def throw_exception():
+                raise HTTPException('This exception message should appear in the logs')
```

Comment:
Minor and optional, but maybe the message text can be defined in a class attribute, so it's easy to visually see where the same text is used/checked against?

Reply:
Good suggestion, because this also turned up an interesting side effect. I originally wrote it assuming that the message of the exception I was also appearing in the logs. But after I tried to implement your suggestions I'm seeing now that only the stack-trace got printed out in `flask_restful_log_exception_error`.

So to change the line to something like `raise HTTPException(expected_error_message)` would also change what the logs show since the stack-trace is showing the code and not the error message itself.

I'm updating offline/main.py now and the test to make sure we get the exception message too.